pkgs/applications/graphics/antimony/default.nix

```diff
@@ -1,29 +1,34 @@
-{ stdenv, fetchFromGitHub, libpng, python3, boost, libGLU_combined, qtbase, ncurses, cmake, flex, lemon }:
+{ stdenv, fetchFromGitHub, libpng, python3
+, libGLU_combined, qtbase, ncurses
+, cmake, flex, lemon
+}:
 
 let
-  gitRev    = "020910c25614a3752383511ede5a1f5551a8bd39";
-  gitBranch = "master";
+  gitRev    = "60a58688e552f12501980c4bdab034ab0f2ba059";
+  gitBranch = "develop";
   gitTag    = "0.9.3";
```

Comment:
`gitTag` can be removed for the time being.

Reply:
It's still used in line 39, I think it's used by the program to display the current version.